Game/multiplayer-project/Assets/Scripts/Http/Models/bot_answer.json

```diff
@@ -0,0 +1,5 @@
+{
+    "message":"It is approximatively 1AM",
+    "action":"harvest",
```

Comment:
action should be a list with exaustive item (action id, location, which one concerned, ...)